src/pineko/theory.py

```diff
@@ -15,11 +15,21 @@
 import rich
 import yaml
 
-from . import check, configs, evolve, parser, theory_card
+from . import check, configs, evolve, parser, scale_variations, theory_card
 
 logger = logging.getLogger(__name__)
 
 
+def check_scvar_evolve(grid, max_as, max_al, tocheck):
```

Comment:
```suggestion
def check_scvar_evolve(grid, max_as, max_al, kind: Scale):
```
A slightly clearer variable name, especially if accompanied by the type hint.

(`type` would be good as well, but `type` is a built-in in Python, and it has a specific meaning in almost any programming language, so I try to avoid it).

Reply:
[459450f](https://github.com/NNPDF/pineko/pull/75/commits/459450f6c5df40023f20d72d1db58623cd164666)